tests/sentry/web/frontend/test_auth_organization_id_login.py

```diff
@@ -0,0 +1,34 @@
+from functools import cached_property
+
+from django.urls import reverse
+
+from sentry.testutils.cases import TestCase
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test
+class AuthOrganizationIdentifierLoginTest(TestCase):
+    @cached_property
+    def organization(self):
+        return self.create_organization(name="test org", owner=self.user)
+
+    @cached_property
+    def path(self):
+        return reverse("sentry-auth-organization-id", args=[self.organization.id])
+
+    def test_redirect_for_logged_in_user(self):
+        self.login_as(self.user)
+        response = self.client.get(self.path)
```

Comment:
```suggestion
        response = self.client.get(self.path, follow=True)
```

For completeness, can you check if the redirects are complete? We'd like to ensure there are no infinite redirect cycles that might occur.

Example for checking that are in this test suite: https://github.com/getsentry/sentry/blob/5a63ff00eae459a14c4192f9b913f93f45337bd5/tests/sentry/web/frontend/test_auth_organization_login.py#L131-L134